Packages/Tracking/Physical Hands/Runtime/Scripts/UI/PhysicalHandsButtonBase.cs

```diff
@@ -0,0 +1,479 @@
+                // Reset hand contact flags if hand is null
+                _leftHandContacting = false;
+                _rightHandContacting = false;
+            }
+
+            // Update overall hand contact flag
+            _contactHandPressing = GetChosenHandInContact();
+        }
+
+        /// <summary>
+        /// Handles actions when a hand exits hovering over the pressable object.
+        /// </summary>
+        /// <param name="hand">The hand exiting hover.</param>
+        protected virtual void OnHandHoverExitPO(ContactHand hand)
+        {
+            // Invoke event for hand hover exit
+            OnHandHoverExit?.Invoke(hand);
+        }
+
+        /// <summary>
+        /// Handles actions when a hand hovers over the pressable object.
+        /// </summary>
+        /// <param name="hand">The hand hovering.</param>
+        protected virtual void OnHandHoverPO(ContactHand hand)
+        {
+            // Invoke event for hand hover
+            OnHandHover?.Invoke(hand);
+        }
+
+        protected virtual void OnCollisionPO(Collision collision)
+        {
+            if (!collision.transform.root.GetComponent<PhysicalHandsManager>())
+            {
+                _objectsContactingButton.Add(collision.gameObject);
+            }
+        }
+
+        protected virtual void OnCollisionExitPO(Collision collision)
+        {
+            _objectsContactingButton.Remove(collision.gameObject);
+        }
+
+        /// <summary>
+        /// Determines whether any chosen hand is in contact with the pressable object.
+        /// </summary>
+        /// <returns>True if a chosen hand is in contact; otherwise, false.</returns>
+        protected bool GetChosenHandInContact()
+        {
+            switch (_whichHandCanPressButton)
+            {
+                case ChiralitySelection.LEFT:
+                    return _leftHandContacting;
+                case ChiralitySelection.RIGHT:
+                    return _rightHandContacting;
+                case ChiralitySelection.BOTH:
+                    return _rightHandContacting || _leftHandContacting;
+                default:
+                    return false;
+            }
+        }
+
+        private void OnDrawGizmosSelected()
+        {
+            if (_pressableObject == null)
+                return;
+
+            Gizmos.color = Color.green;
+
+            Vector3 startPosition = transform.TransformPoint(transform.localPosition + new Vector3(0, _buttonTravelOffset, 0));
+            Vector3 endPosition = transform.TransformPoint((transform.localPosition + new Vector3(0, _buttonTravelOffset, 0)))
+                + transform.up * _buttonTravelDistance;
+
+            Gizmos.color = Color.green; // Y axis
+            Gizmos.DrawLine(startPosition, endPosition);
+            Gizmos.DrawSphere(endPosition, 0.004f); // ButtonStartpoint
+            Gizmos.color = Color.red; 
+            Gizmos.DrawSphere(startPosition, 0.004f); // ButtonEndpoint
+        }
```

Comment:
not sure these are drawn where they should be, nor does it represent the full pressable range (in my testing anyway)

Reply:
I think this is a weird editor glitch, maybe need to look at this further, it does as long as the button offset has been set by clicking on the button object